charts/k8s-monitoring/templates/grafana-agent-events-config.yaml

```diff
@@ -0,0 +1,10 @@
+{{- if and .Values.logs.enabled .Values.logs.cluster_events.enabled }}
```

Comment:
This might have a bug, it appears if you have logs disabled but cluster_events enabled, the configmap doesn't get generated and then the agent-events deployment fails to start. At least in our case we did not want the daemonset logging deployment but we had hoped to collect events with Grafana.

Reply:
Yeah.. it's an inherited mess that I hope to untangle in the future.

the reason cluster_events is inside of logs is because they're processed as log lines and stored in loki. If I could do it again, they'd be at the top level. So `logs.cluster_events.enabled` is the switch for events, `logs.pod_logs.enabled` is the switch for pod logs, and `logs.enabled` is the switch for both.

If you change your config to use:
```yaml
logs:
  pod_logs:
    enabled: false
  cluster_events:
    enabled: true
```

then things should work for you.

Reply:
Eventually, I may remove `logs.enabled` or even move `cluster_events` outside of `logs`, but that'll come with deprecation warnings before I do.